README.md

```diff
@@ -1,2 +1,8 @@
 # any-meeting
 Purpose of this project to waste some time attending different meetings
+
+You can use our fancy webpage to log wasted time during meetings.
+
+You can find it here: http://vgaidarji.me/any-meeting/
+
+![alt tag](https://raw.githubusercontent.com/ilya-gh/any-meeting/gh-pages/doge.jpg)
```

Comment:
@Ilya-Gh Please change `https://raw.githubusercontent.com/ilya-gh/any-meeting/gh-pages/doge.jpg` to `https://raw.githubusercontent.com/vgaidarji/any-meeting/gh-pages/doge.jpg`. As once we merge this the path will be relative to the current repository and not your fork.

cc @thelvis4 @egor-n